.gitignore

```
@@ -19,4 +19,6 @@ docs/venv
 *~
 
 # development
-.vscode
\ No newline at end of file
+.vscode
+*.DS_Store
+/*.DS_Store
```

@VatsalSy I guess you want to exclude "DS_Store" recursively. We can use `**/.DS_Store`
```suggestion
**/.DS_Store
```